### ISSUE TITLE
[RFC] Early/mid game quest improvements

### ISSUE THREAD

**Issue by Cleista** [GTNewHorizons/GT-New-Horizons-Modpack#6297]
#### Which modpack version are you using?
2.0.8.8

#
#### What do you suggest instead/what changes do you propose?
The early game quests and their quest rewards can be tuned to improve quality of life as well as further incentivize players to branch out into mods they otherwise wouldn't touch. Some not very well-known methods of element acquisition can also be made into quests. 

#
### New Quests:

#
#### Concrete (Tier 1 - Steam)
Add optional Steam Age quest: _Tired of walking?_
- **Prerequisites:** Hoppers, Forge Hammer (maybe the automation quests?)
- **Retrieval Task:** 512 Rough Brownstone
- **Retrieval Task:** 512 Brownstone Road
- **Reward:** 16 (or more) Paving Stones of Travel

**Goal:** Encourage players to automate smeltery and build infrastructure to improve QoL when playing with a wider base (which is something that is encouraged by pollution mechanic). 

Paving Stones of Travel are currently mostly irrelevant due to how late unlocking them is and piston boots existing (since they both use the Speed potion effect to boost player speed). Awarding a limited number of them early will help players move faster when in full Hazmat and encourage designing a good base layout that takes advantage of the limited number they are given. 

It also gives players a taste of Thaumcraft utility blocks and can encourage more people to try the mod. 

#
#### RemoteIO Reservoir (Tier 5 - EV)
Adding optional EV quest to craft one: _The sea in a single block_

This infinite water source generates water as fast as you can pump from it, and unlocks in EV but there's no mention of it. This can help notify players. 

#
#### Wireless Connector/Wireless Setup Kit (Applied Energistics)
Adding optional EV quest to craft these: _Basic Wireless AE_

This is part of **AE2 Stuff** and isn't obvious to people who have never used it before, but is extremely useful for automating the cleanroom early on. 

#
#### Ender IO (Tier 3 - HV)
Adding optional EV quest to craft the Ender IO block: _Heard you like Ender IO in your EnderIO_
- **Prerequisites:** Travel Anchor

The EnderIO block is super useful for remotely accessing your ME terminals, chests, drawers, and even some GT machines from travel anchors (or anywhere with a Staff of Traveling). Not a lot of people seem to know about it and adding a quest to craft one should help. 

#
### Quest changes:

#
#### Cough cough - Something needs to be done about the smell (Tier 2 - MV)
Add note to the quest stating that the Advanced Pollution Scrubber loses efficiency per non-air side. 

#
#### We need big toys (Multiblock Goals)
Remove "_Remember to disassemble rotor blades before they break to recover the materials._" since #4970 happened. 

#
#### Purifying the impure (Applied Energistics)
Expand text 
> Purified versions of nether, certus and fluix crystals

to 

> The purified versions of Nether, Certus and Fluix crystals can be grown from seeds assembled from crystal dust and sand. 
> 
> They can be grown in-world for free by dropping in water, or at a much faster rate in a GT autoclave.
> 
> HINT: Distilled water doubles the recipe speed." 

#
#### Welcome to the Church of Benzene (Tier 2 - MV)
For being a church it doesn't do a good job proselytizing. The quest just explains how to get it and makes it sound like a pain in the ass without explaining all of the benefits (Gas Turbine half pollution vs. Combustion Generator, yield, etc.)

Suggest boosting the rewards (e.g. add an Advanced Gas Turbine or give back more Benzene cells) to make players more likely to try it out. 

#
#### Copter Jetpack (Armors and Jetpacks)
Add note to the quest stating that the copterpack pushes dropped items away from you while active. 

I lost about half of the dragon essence and dragon blood from my first dragon kill due to being unaware of this. 

#
#### "The Cook" and "The Dark Wizard" coin purchases (Coins, Coins, Coins)
These coins are currently fairly useless. 

**The Cook:** Suggest adding them to the monster farm quests and allow purchases of some food items (e.g. some mid-tier foods or GT drinks) so that they have a use. 

**The Dark Wizard:** Add some common high demand potions to the shop (Night Vision, Regeneration, Water Breathing, etc.) to give players something to buy with these coins.

**Comment by Prometheus0000** [comment]
First of all, don't put a bunch of stuff in one ticket. If I do only some of it, how am I supposed to mark it down? Also, when I see a bunch of stuff, it goes to the bottom of my list of priorities, since it seems like a huge pain. And it was a huge pain, so there you go.

1) The BS blocks are kinda janky when walking over them, and I don't want to tell people to spend a stack of tin on them either. If you still think it should have a quest, make a new ticket for it.

2) Added quest.

3) There's already a quest.

4) > Not a lot of people seem to know about it

Feel free to describe the intricacies of it (in another ticket), I have no clue.

5) Is this true and can someone confirm it? Again, put it in another ticket.

6) Done.

7) Changed.

8) Boosting rewards is not a good way to convince people to _keep_ using something. Half pollution is also discussed in LV. Added text about byproducts, but if people are too stupid to realize how great it is, that's their fault.

9) Added text.

10) Do this in another ticket.

11) Same.

12) Changed requirement.

13) There's already a quest on the multifarm page.
